console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/ActionDialog.java

```diff
@@ -247,6 +247,26 @@ public void setSubmitButtonState() {
         }
     }
 
+    public String splitTooltipString(String input, int maxLineLength) {
```

Comment:
Please comment the method.
Add parameter validation, what if input is null.
As method is specific provide unit test.
Is this method specific for this class or should be put in utility/common module.
If used only in this class, make it private.